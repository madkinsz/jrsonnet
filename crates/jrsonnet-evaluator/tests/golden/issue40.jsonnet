local conf = {
  n: '',
};

local result = conf {
  assert std.isNumber(self.n) : 'is number',
};

std.manifestJsonEx(result, '')
