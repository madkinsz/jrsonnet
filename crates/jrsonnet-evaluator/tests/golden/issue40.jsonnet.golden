assert failed: is number
   issue40.jsonnet:6:10-31: assertion failure
   issue40.jsonnet:9:1-32:  function <builtin_manifest_json_ex> call